pennylane/pulse/rydberg_hamiltonian.py

```diff
@@ -53,7 +53,7 @@ def rydberg_interaction(
 
     .. seealso::
 
-        :func:`~.rydberg_drive`
+        :func:`~.rydberg_drive`, :func:`rydberg_local_shift`
```

Comment:
is this a relic? The PR looks like there is no explicit `rydberg_local_shift` but rather the detuning term got modified? Some something missing here?

Reply:
Okay, please tag me again when it is cleaned and ready for review 👍